Adventure.java

```
@@ -1,14 +1,16 @@
 import java.util.*;
 
 public class Adventure {
-  public static void main(String[] args) {
+  public static void main (String[] args) {
     Scanner input = new Scanner(System.in);
-    System.out.print("Please enter you name: ");
-    Character player = AdventureData.loadGameDataAndMC(input.nextLine());
-		
-		//Game introduction
-		getIntro(player.getName());
-    
+
+    // Player name and game introduction
+    Character player = playIntro(input);
+
+    if (!AdventureData.loadGameDataAndMC(player)) {
+
```

Is this just error handling? Should we throw an exception or provide a user message here?